app/api/video/migrations/0002_video_delete_time_alter_video_bucket_path_and_more.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.0.2 on 2022-02-27 10:03
```

Comment:
adhere to one migration at a time policy

Reply:
yep, i would like to override base accont model and will need to remove my db and all migrations